app/models/manageiq/providers/openstack/refresh_parser_common/networks.rb

```diff
@@ -164,6 +163,8 @@ def parse_network_port(network_port)
             :fixed_ips                         => network_port.fixed_ips,
             :security_groups                   => security_groups,
           }
+
+          new_result[:name] = network_port.name unless network_port.name.blank?
```

Comment:
Can you verify that no-name will not be a problem in the UI?  It feels like it could get very problematic.  I wonder if the name should default to something unique, like mac address, if a name is not available.